Gruntfile.js

```diff
@@ -7,8 +7,8 @@ module.exports = function(grunt) {
 
 	grunt.initConfig({
 		"clean": {
-			dist: [path.resolve(__dirname, '/polyfills/__dist')],
-			testResults: [path.resolve(__dirname, '/test/results')]
+			dist: [path.resolve(__dirname, './polyfills/__dist')],
+			testResults: [path.resolve(__dirname, './test/results')]
```

Comment:
I _think_ this should just work if the second path doesn't have a preceding slash too. E.g.

```js
path.resolve(__dirname, 'polyfills/__dist');
// rather than
path.resolve(__dirname, './polyfills/__dist')
```

I might be wrong, and it's not a big issue – doesn't _have_ to be changed. Otherwise all good, merge when ready.